packages/augur-ui/package.json

```diff
@@ -50,7 +50,7 @@
     "@ledgerhq/hw-transport-u2f": "4.48.0",
     "@portis/web3": "2.0.0-beta.30",
     "async-es": "2.6.3",
-    "bignumber.js": "6.0.0",
+    "bignumber.js": "8.0.2",
```

Comment:
this seems like a sig. upgrade. Any big changes we need to know about?

Reply:
Nothing that should affect us